tests/open_v2_tests.cpp

```diff
@@ -0,0 +1,103 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <catch2/catch_all.hpp>
+
+struct User {
+    std::string id;
+};
+
+using namespace sqlite_orm;
+
+static const auto default_table = make_table("users", make_column("id", &User::id, primary_key()));
+
+/******************************************************************************/
+/******************************  STATIC CHECKS  *******************************/
+/******************************************************************************/
+
+TEST_CASE("vfs string conversion returns expected strings") {
```

Comment:
We have 'static' assertion tests in the _tests/static_tests_ subfolder. Then you can also omit the `STATIC CHECKS` comment.